docs/modules/deploy/pages/deploying-with-operator.adoc

```diff
@@ -0,0 +1,105 @@
+= Deploy a Cluster with the Enterprise Operator (Preview)
+[[deploying-with-operator]]
+
+NOTE: Hazelcast Enterprise Operator is currently in the early alpha preview stage, but it will soon become the preferred way of installing Hazelcast in Kubernetes/OpenShift environments.
+
+The simplest way of installing and operating your Hazelcast Enterprise cluster in the Kubernetes (and OpenShift) environments is to use Hazelcast Enterprise Operator.
+
+== Deploy Hazelcast Enterprise Operator
+
+You can deploy Hazelcast Enterprise Operator by creating the following `bundle.yaml` file.
+
+[source,yaml]
+----
+include::ROOT:example$/operator/bundle.yaml[]
+----
+
+Then, you can apply it to your Kubernetes cluster with the following command.
+
+[source,shell]
+----
+kubectl apply -f bundle.yaml
+----
+
+At this point, your Hazelcast Enteprise Operator should be up and running. You can check it with the following command.
+
+[source,shell]
+----
+kubectl logs deployment.apps/hazelcast-enterprise-controller-manager
+
+2021-07-14T09:03:40.713Z        INFO    setup   Watching namespace: default
+2021-07-14T09:03:41.524Z        INFO    controller-runtime.metrics      metrics server is starting to listen    {"addr": ":8080"}
+2021-07-14T09:03:41.524Z        INFO    setup   starting manager
+2021-07-14T09:03:41.525Z        INFO    controller-runtime.manager      starting metrics server {"path": "/metrics"}
+...
+----
+
+== Customize Namespace (optional)
+
+By default, Hazelcast Operator is installed into the `default` namespace. If you want to change it, you can either replace all `namespace: default` parameters in `bundle.yaml` or use https://kustomize.io/[Kustomize] to achieve it by creating the following `kustomization.yaml` file.
+
+[source,yaml]
+----
+namespace: <your-namespace>
+resources:
+- bundle.yaml
+----
+
+Then, use Kustomize to apply `bundle.yaml` with the changed namespace.
+
+[source,shell]
+----
+kubectl apply -k .
+----
+
+Note that you'll need to add a parameter `-n <your-namespace>` to all further commands.
+
+== Create Secret with Hazelcast License Key
+
+Hazelcast Enteprise requires having the license key specified as Kubernetes Secret.
```

Comment:
It might be good to mention how to get the license key here. Maybe we can have a link to here: https://docs.hazelcast.com/hazelcast/5.0-beta-1/deploy/using-enterprise-edition.html#requesting-a-license-key

Reply:
Added